modules/audit-logging/src/Volo.Abp.AuditLogging.Domain/Volo/Abp/AuditLogging/AuditingStore.cs

```diff
@@ -43,6 +43,9 @@ public async Task SaveAsync(AuditLogInfo auditInfo)
             catch (Exception ex)
             {
                 Logger.LogException(ex, LogLevel.Error);
+
+                if (throwExceptions)
```

Comment:
Instead of throwing a new exception, just throw current one, like:

````csharp
throw
````

However, it is better to not use try-catch if throwExceptions is false.

Reply:
shouldn't it log exception if hideErrors =false ?

Reply:
No need to logging, because this exception will be caught by another code (exception filter) and logged there.